dev-python/cadquery/cadquery-2.2_rc20220506.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python parametric CAD scripting framework based on OCCT"
+HOMEPAGE="https://cadquery.readthedocs.io"
+
+# The official CadQuery 2.1 tarball requires the obsolete OCCT 7.4.0, but
+# CadQuery 2.2 has yet to be officially released. We instead package a commit
+# known to work as expected with OCCT 7.5.2.
+MY_COMMIT="803a05e78c233fdb537a8604c3f2b56a52179bbe"
+
+#FIXME: Uncomment on bumping to the next stable release.
+# SRC_URI="https://github.com/CadQuery/cadquery/archive/refs/tags/${PV}.tar.gz"
+SRC_URI="https://github.com/CadQuery/cadquery/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Dependencies are intentionally listed in "conda/meta.yml" order. Due to its
+# Anaconda focus, "setup.py" currently fails to list dependencies.
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/docutils[${PYTHON_USEDEP}]
+			dev-python/path-py[${PYTHON_USEDEP}]
+		')
+	)
+"
+RDEPEND="
+	>=dev-python/cadquery-ocp-7.5.0[${PYTHON_SINGLE_USEDEP}]
+	$(python_gen_cond_dep '
+		>=dev-python/pyparsing-2.0.0[${PYTHON_USEDEP}]
+		dev-python/ezdxf[${PYTHON_USEDEP}]
+		dev-python/ipython[${PYTHON_USEDEP}]
+		dev-python/typing-extensions[${PYTHON_USEDEP}]
+		dev-python/nptyping[${PYTHON_USEDEP}]
```

Comment:
It looks like we are missing one other dependency here:
```
ImportError while importing test module '/var/tmp/portage/dev-python/cadquery-2.2_rc20220506/work/cadquery-803a05e78c233fdb537a8604c3f2b56a52179bbe/tests/test_workplanes.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/site-packages/_pytest/python.py:608: in _importtestmodule
mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/lib/python3.10/site-packages/_pytest/pathlib.py:533: in import_path
importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
???
<frozen importlib._bootstrap>:1027: in _find_and_load
???
<frozen importlib._bootstrap>:992: in _find_and_load_unlocked
???
<frozen importlib._bootstrap>:241: in _call_with_frames_removed
???
<frozen importlib._bootstrap>:1050: in _gcd_import
???
<frozen importlib._bootstrap>:1027: in _find_and_load
???
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
???
<frozen importlib._bootstrap>:688: in _load_unlocked
???
<frozen importlib._bootstrap_external>:883: in exec_module
???
<frozen importlib._bootstrap>:241: in _call_with_frames_removed
???
tests/__init__.py:1: in <module>
from cadquery import *
cadquery/__init__.py:14: in <module>
from .occ_impl import exporters
cadquery/occ_impl/exporters/__init__.py:10: in <module>
from ...cq import Workplane
cadquery/cq.py:61: in <module>
from .sketch import Sketch
cadquery/sketch.py:19: in <module>
from typish import instance_of, get_type
E   ModuleNotFoundError: No module named 'typish'
```
This package recently got last-rited from ::gentoo, but we can probably easily bring it back in ::sci.
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=0033be122a2c05126b66085e0c9c6b107c0410ff

Reply:
That's why I `emerge --sync` only once a year! (/s)

Will take a look

Reply:
Added the ebuild.